packages/grafana-ui/src/components/Monaco/theme.ts

```diff
@@ -0,0 +1,23 @@
+import { GrafanaTheme2 } from '@grafana/data';
+import { Monaco } from './types';
+
+export default function defineThemes(monaco: Monaco, theme: GrafanaTheme2) {
+  const colors = {
+    'editor.background': theme.components.input.background,
+    'minimap.background': theme.colors.background.secondary,
```

Comment:
I wasn't sure whether this should be primary or secondary for the minimap.

Here's primary for comparison:

![chrome_AjmSom5Fat](https://user-images.githubusercontent.com/46142/117060173-8a893400-ad18-11eb-9a03-d111bf56c02a.png)

I thought it was basically the same as the overlay bg so blended in too much?